index.html

```diff
@@ -795,18 +795,20 @@ <h2>
         The user agent MUST navigate the application context as per [[!HTML]]'s
         <a>navigate algorithm</a> with exceptions enabled. If the URL of the
         resource being loaded in the navigation is not <a>within scope</a> of
-        the navigation scope, then the user agent MUST behave as if the
-        application context is not <a>allowed to navigate</a> (this provides
-        the ability for the user agent to perform the navigation in a different
-        browsing context, or in a different user agent entirely). If during the
-        handle redirects step of HTML's <a>navigate algorithm</a> the redirect
-        URL is not <a>within scope</a>, abort HTML's navigation algorithm with
-        a <code>SecurityError</code>.
+        the navigation scope of the application context's manifest, then the
+        user agent MUST behave as if the application context is not <a>allowed
+        to navigate</a> (this provides the ability for the user agent to
```

Comment:
I am wondering whether we should do the () part without ()'s and with uppercase This

Reply:
Done, but note I am about to rewrite this sentence in the next PR (changing default scope) anyway.